src/backend/src/services/tasks.services.ts

```diff
@@ -51,9 +55,20 @@ export default class TasksService {
     const isProjectLeadOrManager =
       createdBy.userId === requestedWbsElement.projectLeadId || createdBy.userId === requestedWbsElement.projectManagerId;
 
-    if (!isLeadership(createdBy.role) && !isProjectLeadOrManager && !teams.some((team) => isUserOnTeam(team, createdBy))) {
+    const curWorkPackages = project.workPackages;
+
+    const isWorkPackageLead = curWorkPackages.some((workPackage) => {
```

Comment:
I know it doesn't say in the ticket but we should also allow WorkPackageManagers to create tasks as well (in both places)